content/guides/models/app/features/panels/_index.md

```diff
@@ -69,30 +69,33 @@ To get back to a panel's workspace from full-screen mode, click the left-pointin
 
 ## Add panels
 
-You can add panels to your workspace, either globally or at the section level.
+This section shows various ways to add panels to your workspace.
 
-To add a panel:
+### Quick add panels {#quick-add}
 
-1. To add a panel globally, click **Add panels** in the control bar near the panel search field.
-1. To add a panel directly to a section instead, click the section's action `...` menu, then click **+ Add panels**.
-1. Select the type of panel to add.
-   
-   {{< img src="/images/app_ui/add_single_panel.gif" >}}
+Use **Quick add** to generate a standard panel for each key you select, either globally or at the section level.
 
-### Quick add
-
-**Quick Add** allows you to select a key in the project from a list to generate a standard panel for it.
+{{% alert %}}
+For an automated workspace with no deleted panels, the **Quick add** option is not visible becasue the workspace already includes standard panels for all logged keys. You can use **Quick add** to re-add a panel that you deleted.
```

Comment:
typo ☺️🙈

```suggestion
For an automated workspace with no deleted panels, the **Quick add** option is not visible because the workspace already includes standard panels for all logged keys. You can use **Quick add** to re-add a panel that you deleted.
```